sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -274,12 +276,35 @@ class MapboxMap :
     onMapLoadErrorListener: OnMapLoadErrorListener? = null,
   ) {
     checkNativeMap("loadStyle")
-    this.loadStyleUri(
-      styleExtension.styleUri,
-      transitionOptions,
-      { style -> onFinishLoadingStyleExtension(style, styleExtension, onStyleLoaded) },
-      onMapLoadErrorListener
+    initializeStyleLoad(
+      onStyleLoaded,
+      styleDataStyleLoadedListener = { style ->
```

Comment:
if a new property (feature) is added to the style, when it would be 'bound' to the style? For example, `+model(...)`.

Reply:
Hmm we don't have +model apis in the style extension

Reply:
Honestly we have to remember to bind everything for now. Though about adding a comment - but again, one will need to read this comment to not forget to bind new feature.